backend/routes/user.js

```diff
@@ -0,0 +1,124 @@
+const express = require('express');
+const router = express.Router();
+const gravatar = require('gravatar');
+const bcrypt = require('bcryptjs');
+const jwt = require('jsonwebtoken');
+const passport = require('passport');
+const validateRegisterInput = require('../validation/register');
+const validateLoginInput = require('../validation/login');
+
+const User = require('../models/User');
+
+router.post('/register', function(req, res) {
+
+  const {
+    errors,
+    isValid
+  } = validateRegisterInput(req.body);
+
+  if (!isValid) {
+    console.log("error in 20 lines of code user.js")
```

Comment:
?

Reply:
if you about that console log ,I have already deleted it